static/docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+# Troubleshooting
+
+In this section we provide help for some of the problems that DVC user might
+stumbe upon.
```

Comment:
some of the problems -> some special situations ?

Reply:
I don't think we should avoid using strong wording, this is a problem from a point of the user view. I would say that a `special situation` does not exactly express the severeness of this. I do understand that might change if later we discover some less severe troubles to shoot, but we can then adjust. What do you think?

Reply:
Yes, as you prefer. I'm not about this suggestion.